data/presets/amenity/vending_machine/fuel.json

```diff
@@ -4,6 +4,12 @@
         "point",
         "vertex"
```

Comment:
Do we need to have them as vertex? I haven't seen fuel pumps that are attached to walls (or even fuel stations that have walls on the outside part).

Reply:
Looks like there are (https://community.openstreetmap.org/t/differing-approaches-to-tagging-vending-machines-for-motor-fuel-dispensing-have-caused-confusion/103421/1)